scrapy_poet/backend.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import scrapy
+from scrapy.utils.defer import maybe_deferred_to_future
+from web_poet import HttpRequest
+from web_poet.exceptions import (
+    HttpError,
+    HttpRequestError,
+)
+
+from scrapy_poet.utils import (
+    http_request_to_scrapy_request,
+    scrapy_response_to_http_response,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def create_scrapy_backend(backend):
```

Comment:
Would it make sense to call the argument as something like `download_func`, not `backend`?